.github/workflows/unit-test.yml

```diff
@@ -154,3 +154,42 @@ jobs:
       - name: Run unit tests
         run: |
           go test -v ./internal/querytest/...
+
+  scripts-test:
+    name: Run unit tests for scripts
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      checks: write
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # v5.0.0
+        with:
+          persist-credentials: false
+
+      - name: Setup Node.js
+        uses: actions/setup-node@2028fbc5c25fe9cf00d9f06a71cc4710d4507903 # v4.0.4
+        with:
+          node-version: "18"
```

Comment:
That's a rather old version of node, [which is end-of-life](https://nodejs.org/en/about/previous-releases) - is there a reason we're using that instead of a more recent LTS release?